exporter/awsemfexporter/metric_translator.go

```diff
@@ -55,15 +54,22 @@ type rateState struct {
 	timestamp int64
 }
 
-// CWMetrics defines
-type CWMetrics struct {
-	Measurements []CwMeasurement
+// MetricInfo defines
+type MetricInfo struct {
+	Value  interface{}
+	Unit   string
+}
+
+// GroupedMetric defines
+type GroupedMetric struct {
+	Namespace    string
 	Timestamp    int64
-	Fields       map[string]interface{}
```

Comment:
In our latest discussion, Labels should be defined as map[string]string

Reply:
Right now defined as interface{}, will change accordingly